src/assignments/tech-jobs-mvc.rst

```diff
@@ -0,0 +1,443 @@
+.. _tech-jobs-mvc:
+
+Assignment #3: Tech Jobs (MVC Edition)
+======================================
+
+Introduction
+------------
+
+Your first two tasks as an apprentice went well! You, Blake, and Sally built
+the TechJobs console prototype and then refactored the code to move it to an
+object-oriented format.
+
+After demonstrating the prototype for the Company Team at LaunchCode, it
+received the green light to be fully built out as a web application.
+
+.. index:: ! minimum viable product
+
+The first step in this process will be to quickly develop a `minimum viable
+product <https://en.wikipedia.org/wiki/Minimum_viable_product>`__, or MVP. The
+goal is to get a functioning web app up and running with as little work as
+possible. That way, additional feedback and testing can be done early in the
+development process. After that, additional behind-the-scenes work will be
+carried out to fully develop the model and data side of the application.
+
+For this next step in the project, you’ll be working with Carly.
+
+.. figure:: figures/LC-Carly.png
+   :scale: 55%
+   :alt: Carly's LaunchCode avatar.
+
+Carly was once a LaunchCode apprentice as well, so she knows just what
+it’s like to be in your shoes. She’s done some initial work on the
+project and left you some ``TODO`` tasks that she knows you can handle.
+
+Learning Objectives
+-------------------
+
+In this project, you’ll show that you can:
+
+#. Read and understand code written by others.
+#. Work within the controller and view portions of an ASP.NET application.
+#. Use Razor syntax to display data within a view.
+#. Create new handler methods to process form submission.
+
+TechJobs (MVC Edition)
+----------------------
+
+You’ll start with some code that Carly has provided. The idea behind your
+current assignment is to quickly deliver a functioning ASP.NET application,
+so you’ll focus on the controllers and views.
+
+In order to do this, you’ll be reusing the ``JobData`` class and
+``job_data.csv`` file from the console app. You will eventually have to go back
+and rewrite the data portion of the application to make a true, database-backed
+model. However, using the existing ``JobData`` class to provide some basic data
+functionality lets you focus on the views and controllers for now.
+
+Your Assignment
+---------------
+
+The list below provides a general overview of your assigned tasks. Specific
+details for each part appear in the following sections, so be sure to read them
+carefully as you solve each problem.
+
+#. Review Carly’s code in the ``JobData`` file as well as in the existing
+   controllers and views.
+#. As her next step, Carly created two basic templates to show lists of jobs
+   for each field of the data (``employer``, ``location``, etc.). You will need
```

Comment:
Minor point of confusion here: is this referring to form fields or class properties when it talks about fields of data?